streaming/text/convert/README.md

```diff
@@ -2,16 +2,71 @@
 
 To use Streaming Dataset, We first convert the dataset from its native format to MosaicML's streaming dataset format (a collection of binary `.mds` files). Once in `.mds` format, we can store the dataset in a central location (filesystem, S3, GCS, etc.) and stream the data to any compute cluster, with any number of devices, and any number of CPU workers, and it all just works.
 
-## Converting C4 to streaming dataset .mds format
+Follow the below steps to convert the Natural Language Processing dataset into a streaming MDS format. Also checkout the supported [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for advanced usage.
```

Comment:
Here we refer to "MDS" as a name but above we refer to it as a ".mds" file extension.  I know it's both but it we should be consistent.  I recommend using "MDS" when discussing it as a format and keeping the ".mds" extension to code snippets and examples.

Reply:
```suggestion
Check out steps below for information on converting common NLP datasets to MDS format.  Please see [MDSWriter()](https://streaming.docs.mosaicml.com/en/latest/api_reference/generated/streaming.MDSWriter.html) parameters for details on advanced usage.
```